doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -479,19 +479,21 @@ toplevel. This kind of file is called a *script* for Coq. The standard
 (and default) extension of Coq’s script files is .v.
 
 
-.. cmd:: Load {? Verbose } {| @string | @ident }
+.. cmd:: Load {? Verbose } {| @string | @qualid }
 
-   Loads a file.  If :n:`@ident` is specified, the command loads a file
-   named :n:`@ident.v`, searching successively in
-   each of the directories specified in the *loadpath*. (see Section
-   :ref:`libraries-and-filesystem`)
+   Executes the commands of a file in the current context.
+
+   If :n:`@qualid` is specified, :n:`Load` searches
+   in the *loadpath* (see Section :ref:`libraries-and-filesystem`)
+   for a file with suffix :n:`.v` whose logical name is :n:`@qualid`.
 
    If :n:`@string` is specified, it must specify a complete filename.
    `~` and .. abbreviations are
    allowed as well as shell variables. If no extension is specified, Coq
-   will use the default extension ``.v``.
+   will use the default extension ``.v``. If the filename is relative,
+   it is searched in the loadpath, ignoring the logical part.
```

Comment:
```suggestion
   appends the default extension ``.v``. If the filename is relative,
   Coq searches in the loadpath, ignoring the logical part of :n:`@qualid`.
```

"ignoring the logical part" -- You mean (for example) strip off `Coq.` from `Coq.Init.Nat`.  Can only be the first component, maybe can say that more explicitly?